app/javascript/components/StudyResults.js

```diff
@@ -1,30 +1,46 @@
 import React from 'react'
+import Tab from 'react-bootstrap/lib/Tab'
+import Tabs from 'react-bootstrap/lib/Tabs'
 import { useTable, usePagination } from 'react-table'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import {
-  faAngleDoubleLeft, faAngleLeft, faAngleRight, faAngleDoubleRight
-} from '@fortawesome/free-solid-svg-icons'
 import Study from './Study'
-
+import PagingControl from './PagingControl'
 /**
  * Wrapper component for studies on homepage
  */
-const StudyResults = ({ results, changePage }) => {
+export default function StudyResultsContainer(props) {
```

Comment:
The default export by convention should be named the same as the file. 
FYI: we're getting rid of the tab container for now--we'll put it back when we implement the 'files' tab